simple_rl/mdp/MDPClass.py

```diff
@@ -46,6 +46,9 @@ def set_gamma(self, new_gamma):
     def set_step_cost(self, new_step_cost):
         self.step_cost = new_step_cost
 
+    def is_in_goal_state(self):
```

Comment:
Good point, Dave. Any suggestions on how I can modify the BSS solver so that it doesn't need access to this function? My first thought is to use the test if `state.is_terminal()`, but in the 1D Maze POMDP, the goal state is not terminal. So in general our solvers would need a way to know when to terminate -- not sure where that would come from.

Reply:
Nvm by choosing to terminate at the goal state, BSS is implicitly treating the goal state as a terminal state. So solvers should just check if the `state.is_terminal()` and leave it up to the domain to define which states it considers terminating.